index.html

```diff
@@ -12308,21 +12323,29 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
             </caption>
             <thead>
               <tr>
-                <th scope="col">Value</th>
+                <th scope="col">Value (keyword)</th>
+                <th scope="col">State</th>
                 <th scope="col">Description</th>
               </tr>
             </thead>
             <tbody>
               <tr>
-                <th class="value-name" scope="row">false</th>
+                <th class="value-name" scope="row">false, empty string ("")</th>
+                <th class="value-name" scope="row">False</th>
                 <td class="value-description">Assistive technologies will present only the changed node or nodes.</td>
               </tr>
               <tr>
                 <th class="value-name" scope="row">true</th>
+                <th class="value-name" scope="row">True</th>
                 <td class="value-description">Assistive technologies will present the entire changed region as a whole, including the author-defined label if one exists.</td>
               </tr>
             </tbody>
           </table>
+          <p>
+            For <a data-cite="html/common-dom-interfaces.html#reflect">reflection</a> purposes, the attribute's
+            <a data-cite="html/common-microsyntaxes.html#missing-value-default">missing value default</a> is the Undefined state (null) that has no associated keyword, and its
+            <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> is the False state.
```

Comment:
As the `False` state has two keywords we will need to specify the canonical keyword for this state;

```suggestion
            <a data-cite="html/common-microsyntaxes.html#invalid-value-default">invalid value default</a> is the False state. For the purposes of reflection, the canonical keyword for the False state is the `false` keyword.
```

(You might want to link to "reflection", "canonical keyword", etc).

This is the same for other properties like `aria-busy`, `aria-checked` and so on - I won't leave a note for each.

Reply:
https://github.com/whatwg/html/pull/11303 has landed so we can consider this resolved, I believe.